### PR TITLE
Fix for potential progress loss when changing languages

### PR DESCRIPTION
**Pull request by danielghost**
https://github.com/adaptlearning/adapt_framework/issues/3139.

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]
👀

File: js/languagePickerDrawerView.js [hunk 1]
```diff
@@ -67,6 +66,12 @@ define([
       Adapt.trigger('drawer:closeDrawer');
     },
 
+    onPopupOpened: function () {
+      // move popup close focus to #focuser
+      Adapt.a11y._popup._focusStack.pop();
```
Comment (oliverfoster) [inline js/languagePickerDrawerView.js:71]: https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/a11y.js#L721

File: js/languagePickerDrawerView.js [hunk 2]
```diff
@@ -67,6 +66,12 @@ define([
...
+      Adapt.a11y._popup._focusStack.push($('#a11y-focuser'));
```
Comment (oliverfoster) [inline js/languagePickerDrawerView.js:72]:
```suggestion
      Adapt.a11y.setPopupCloseTo($('#a11y-focuser'));
```
https://github.com/adaptlearning/adapt_framework/blob/c0e308f5302056bc8a48dfd055d6f1669a7ba944/src/core/js/a11y.js#L721

https://github.com/adaptlearning/adapt_framework/blob/c0e308f5302056bc8a48dfd055d6f1669a7ba944/src/core/js/a11y/popup.js#L213-L215

**Review by oliverfoster — APPROVED** [review]

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**